projects/wmf_default.yaml

```diff
@@ -126,9 +126,11 @@ paths:
                         Gumagamit:Lsjbot/Kartrutor2: true
                       commons.wikimedia.org:
                         Commons:Quality_images_candidates/candidate_list: true
+                        Commons:Quality_images/Subject/Places/Natural_structures: true
                         User:Darkweasel94/Vienna/2016_December_14-16: true
                         User:J_budissin/Uploads/BiH/2016_December_11-20: true
                         User:Jarekt/d: true
+                        User:Nicolas_Rück_(WMDE)/Supported_by_Wikimedia_Deutschland_2015/1: true
```

Comment:
Oh, indeed, sorry. I've made a little script that analyzes kafka error topic over the last week and gives out the most problematic pages (with  the highest number of 500/504 errors) and these 2 are definitely problematic. Will update to include /2 as well.